docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     environment:
       FIDESLOG__LOGGING_LEVEL: "debug"
       FIDESLOG__SERVER_HOST: "fideslog"
-      FIDESLOG__SERVER_HOT_RELOAD: true
+      FIDESLOG__SERVER_HOT_RELOAD: "true"
```

Comment:
using just `true` caused isort to fail with:

```
Running isort checks...
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.fideslog.environment.FIDESLOG__SERVER_HOT_RELOAD contains true, which is an invalid type, it should be a string, number, or a null
```

I've confirmed using string `"true"` and `"false"` here are valid and functional using `make api`